src/text-field.jsx

```diff
@@ -55,6 +55,7 @@ const TextField = React.createClass({
     rowsMax: React.PropTypes.number,
     style: React.PropTypes.object,
     type: React.PropTypes.string,
+    underline: React.PropTypes.bool,
```

Comment:
maybe `underlineVisible`? `underline` feels like you need to pass the "Underline" node through this prop. @oliviertassinari Thoughts?